website/static/js/osfHelpers.es6.js

```diff
@@ -372,6 +372,50 @@ var decodeText = function(text) {
     return text;
 };
 
+var parseName = function(text) {
+    const suffixList = ['jr','jnr','sr','snr','dr','phd','i','ii','iii','iv','v','md','esq'];
```

Comment:
Ahh, yea in java an array is an implementation of list so I got used to naming them like that haha, I'll add a comment describing its purpose